packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -111,19 +111,16 @@ const EmbedPreview = ( {
 			disabled={ ! isSelected }
 		>
 			<View>
-				{
-					// eslint-disable-next-line no-undef
-					__DEV__ && previewable ? (
```

Comment:
The inline preview feature is no longer limited to development mode so after this PR gets merged the feature will be available in production for [the specified providers](https://github.com/WordPress/gutenberg/blob/f879a9d53b6b268b794ead72907adf72c2830885/packages/block-library/src/embed/edit.native.js#L35-L37).